generate-routes.ts

```diff
@@ -31,8 +31,11 @@ async function main(): Promise<void> {
   ])
 }
 
+const openapiResponseKeyProp = 'x-fern-sdk-return-value'
+
 const routePaths = [
   '/access_codes',
+  '/access_codes/simulate',
```

Comment:
We need to add this under `routePathSubresources`

Reply:
https://github.com/seamapi/javascript-http/pull/87/commits/98b8d5de0ff2d05ebda402bd9c7c8a920bd50933